docs/modules/ROOT/pages/index.adoc

```diff
@@ -2,4 +2,22 @@
 
 nfs-client-provisioner is a Commodore component to manage nfs-client-provisioner.
 
-See the xref:references/parameters.adoc[parameters] reference for further details.
+See the xref:references/parameters.adoc[parameters] reference for details on
+configurable options.
+
+== Prerequisites
+
+The following packages have to be installed on the host system:
+
+RHEL / CentOS
+
+```
```

Comment:
That is markdown, not Asciidoc. Instead of code blocks, I would do this with a definition list.
```
RHEL / CentOS::
  nfs-utils

Ubuntu / Debian::
   nfs-common
```